### DISCUSSION TITLE
Usb not working

### DISCUSSION THREAD

**Discussion by CocoJambo4978** [termux/termux-packages#17639] [General]
![Screenshot_2023-08-14-18-52-31-370_com termux](https://github.com/termux/termux-packages/assets/142255516/e44389e7-21cf-4e88-8a60-299486ab114b)

**Comment by sylirre** [comment]
Do you have Termux:API **application** installed? The package `termux-usb` doesn't work without this one.

Link to the Termux:API: https://f-droid.org/packages/com.termux.api/
